src/lib/caisy/content-entry/import.ts

```diff
@@ -0,0 +1,165 @@
+import { db } from "../../common/db";
+import {
+  initSdk,
+  PutManyDocumentFieldLocalesRequestInput,
+  DocumentFieldLocaleUpsertInputInput,
+  DocumentWithFieldsInput,
+  DocumentFieldInput,
+  ReferenceType,
+  DocumentFieldLocaleChangeSetFragmentDoc,
+  DocumentFieldLocaleChangeSet,
+} from "@caisy/sdk";
+import { CaisyRunOptions } from "../provider";
+import {
+  contentEntry,
+  contentEntryField,
+  contentLocale,
+  contentType,
+  contentTypeField,
+  contentTypeGroup,
+} from "../../common/schema";
+import {
+  denormalizeCaisyContentTypeVariant,
+  denormalizeCaisyContentEntryStatus,
+  denormalizeCaisyFieldEntry,
+  fromStringToCaisyContentFieldType,
+  fromStringToCaisyBlueprintFieldType,
+} from "./normalize";
+import { time } from "console";
+import { processDataForCaisyDocumentField, ContentEntryFieldData } from "../../common/types/content-entry";
+
+async function fetchDocumentsFromDatabase({ sdk, projectId, onProgress, onError }: CaisyRunOptions): Promise<void> {
+  const documentLocalesInputs: DocumentFieldLocaleUpsertInputInput[] = [];
+  const documentInputs: DocumentWithFieldsInput[] = [];
+  const documentLocaleRows = await db.select().from(contentLocale).execute();
+  const documentRows = await db.select().from(contentEntry).execute();
```

Comment:
@gkoutzioulis i think here and for the documentFields we can not fetch ALL documents at once if the dataset is larger (imagine you have 100000 documents) 

Maybe best to create a page size of 1000 documents at once or so 

-> Sort the documents and document fields by documentId and paginate through them would make sense for me

Reply:
for locales its fine to fetch them all at once because there might be just a few

Reply:
i think we also have a limit on PutManyDocuments in caisy

Reply:
at least if the request is larger then 50mb it will fail 

so maybe 100 documents at a time is better then 1000